superset-frontend/src/explore/components/controls/ColorSchemeControl/index.jsx

```diff
@@ -196,7 +179,7 @@ export default class ColorSchemeControl extends React.PureComponent {
     };
 
     return (
-      <Select
+      <StyledSelect
```

Comment:
I'm noticing a weird thing where the selected color starts at the top of the list, but then moves to another location if you move the mouse outside the list and back in.  I don't see how this PR would have caused that problem, but if it's easy to fix maybe we could fix it here?

https://user-images.githubusercontent.com/13007381/184456288-dd7dcf96-c3ed-4140-b891-5658cfa6b2fb.mov

Reply:
This is a Select component issue that it makes sense to fix in a separate PR. I'll open an issue for this problem. Thanks for catching it!